custom/ewsghana/alerts/alerts.py

```diff
@@ -66,7 +66,7 @@ def on_going_process_user(user, test=False):
             send_sms_to_verified_number(verified_number, message)
         else:
             send_test_message(verified_number, message)
-        if user.email:
+        if user.email and verified_number.backend_id == 'MOBILE_BACKEND_TWILIO':
```

Comment:
@czue I don't see option to set email backend to user but I think this solution for now prevent from sending emails to real users. What do you think?